examples/vpc_service_control/vpc_sc_project/variables.tf

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+variable "project_id" {}
+
+variable "service_account_file" {}
+
+variable "organization_id" {}
+
+variable "billing_account_id" {}
+
+variable "cloud_router_region" {}
+
+variable "ip_addr_of_onprem_vpn_router" {}
```

Comment:
With or without the commenting and uncommenting workflow, I don't think this variable is necessary nor is the output if the same name. 
```suggestion
```

Reply:
We need this so the on-prem and VPC SC  VPN routers can find each other.

Reply:
You're right I must have been looking in the wrong folder.